search/includes/classes/class-health.php

```diff
@@ -315,6 +360,33 @@ public static function filter_expected_post_rows( $rows, $post_types, $post_stat
 		return $filtered;
 	}
 
+	public static function simplified_diff_document_and_prepared_document( $document, $prepared_document ) {
+		$ignored_keys = array(
```

Comment:
Can we make this config reusable so that both diff functions (simple and full) always use the same fields?